compiler/src/iree/compiler/Dialect/Flow/Transforms/AnnotateDispatches.cpp

```diff
@@ -267,6 +272,19 @@ static std::string summarizeLinalgExtOp(Operation *op) {
     mainTensor.getElementType().print(sstream);
     sstream.flush();
   }
+
+  // append fused consumer (`linalg` or `linalg_ext`)
+  // e.g ..._1xDxDx1xf16_linalg_generic
+  auto users = op->getUsers();
+  if (!users.empty() && std::next(users.begin()) == users.end()) {
+    auto user = *users.begin();
+    auto userOpName = user->getName().getStringRef();
+    auto pos = userOpName.find('.');
+    assert(pos != StringRef::npos && "expected dialect name");
+    suffix += "_";
+    suffix.insert(suffix.end(), userOpName.begin() + pos + 1, userOpName.end());
```

Comment:
I think we can reuse `getOpNameWithoutDialectName` here, so you don't need to deal with `.` yourself.